scripts/enemy/EnemySpawner.gd

```diff
@@ -0,0 +1,16 @@
+extends Node3D
+
+
+@onready var enemies_node = $"../../Enemies"
```

Comment:
Reference possible issue

Reply:
We need to be careful about the Enemies node. It needs to be present outside the Player node